src/pages/workspaces/workspace/WorkspaceMenu.js

```diff
@@ -61,25 +61,25 @@ const WorkspaceMenuContent = ({ canShare, isAzureWorkspace, isLocked, isOwner, o
 
   return h(Fragment, [
     h(MenuButton, {
-      disabled: isAzureWorkspace,
+      disabled: !workspaceLoaded || isAzureWorkspace,
       tooltip: workspaceLoaded && isAzureWorkspace && 'Cloning is not currently supported on Azure Workspaces',
       tooltipSide: 'left',
       onClick: onClone
     }, [makeMenuIcon('copy'), 'Clone']),
     h(MenuButton, {
-      disabled: !canShare || isAzureWorkspace,
+      disabled: !workspaceLoaded || !canShare || isAzureWorkspace,
```

Comment:
For the benefit of other reviewers: I added checks for `!workspaceLoaded` on all of these for clarity. But because the default value of `canShare` and `isOwner` is falsey before the initial Ajax response is received, these menus were already correctly disabled during the pre-loaded period.